### PR TITLE
Apply libunwind fix

### PR DESCRIPTION
**Pull request by al13n321**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

https://github.com/ClickHouse/libunwind/pull/30

### CODE REVIEW

**Comment by robot-ch-test-poll3** [comment]
*This is an automated comment for commit 68eea21f1f3d69a975392d71775f74328447c2a3 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/66977/68eea21f1f3d69a975392d71775f74328447c2a3/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Docs check</td><td>Builds and tests the documentation</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/91693daf599bbaeaed5adb8cba257c176628e2be/docs_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

**Review by antonio2368 — APPROVED** [review]
Please update the reference to commit in master of libunwind contrib once you are happy
This change doesn't break anything I checked. :+1:

File: src/Common/QueryProfiler.cpp [hunk 1]
```diff
@@ -57,7 +57,7 @@ namespace
 
         auto saved_errno = errno;   /// We must restore previous value of errno in signal handler.
 
-#if defined(OS_LINUX)
+#if defined(OS_LINUX) && false //asdqwe
```
Comment (antonio2368) [inline src/Common/QueryProfiler.cpp:60]: Why was it merged with these changes?

Comment (al13n321) [reply src/Common/QueryProfiler.cpp:60]: Sorry. https://github.com/ClickHouse/ClickHouse/pull/67152

**Pull request merged**